packages/ui/src/components/Badge/types.ts

```diff
@@ -0,0 +1,10 @@
+export type BadgeType = 'active' | 'error' | 'warning' | 'info' | 'debug' | 'pending'
+
+export enum BadgeTypes {
+  active,
+  error,
+  warning,
+  info,
+  debug,
+  pending,
+}

```

Comment:
Unnecessary duplication and enum `BadgeTypes` is useless because its values are numbers and not actual strings.

Reply:
https://www.typescriptlang.org/play?#code/KYOwrgtgBAQghgEwObACoE8AOwoG8CwAUFCVHAMYAuAlgG7AA0RpU1IFN9RAvkA